setup.py

```diff
@@ -39,6 +39,9 @@
         "osh/hub/scripts/osh-stats",
         "osh/worker/osh-worker",
     ],
+    "/usr/share/licenses/osh": [
```

Comment:
I do not think we need this in `setup.py`.  You can install the license file in the spec file directly.  See e.g. `curl.spec`: https://src.fedoraproject.org/rpms/curl/blob/f9311ae69d7c143fec8f3282907ac95546869cfc/f/curl.spec#_378

Reply:
Shall I leave it as it is, if somebody may choose to have a non-rpm package for this in the future?

Reply:
I guess that non-rpm packages have their own ways to handle license files.  This looks rather Fedora specific.